### DISCUSSION TITLE
Planning: Q12022 Roadmap call! 11/13 January 2022

### DISCUSSION THREAD

**Discussion by emckean** [google/docsy#799] [General]
Hi Docsy folks! We're planning a roadmap/community call via Google Meet in January.

We have two possible dates: 

~🚀  11 January (Tuesday), 8-9 am PST [timezone planner link](https://www.timeanddate.com/worldclock/meetingtime.html?iso=20220111&p1=224&p3=78)~
🎉  13 January (Thursday), 9-10 am PST [timezone planner link](https://www.timeanddate.com/worldclock/meetingtime.html?month=1&day=13&year=2022&p1=224&p2=78&iv=0)

We'll be discussing our plans for 2022! We'll add an agenda to this discussion closer to the meeting date, but not later than January 7, 2022.

To signal your preferred date/time, please emoji-react to this message with 🚀  for 11 January and 🎉  for 13 January. :) We hope to confirm the time before the end of 2021, so please react by midnight UTC 27 December. 

Looking forward to the call!

**Comment by emckean** [comment]
Agenda for this week's meeting (13 January (Thursday), 9-10 am PST)

* Introductions, please add yours to the chat (not via voice)
* Governance update, hi to the PSC!
* Roadmap overview & 2022 plans
* Q&A
* How to be involved 

To get an invite to the Google meet, please join the [docsy-users](https://groups.google.com/g/docsy-users) group (if you cannot join a group, please ping @emckean directly)

**Comment by chalin** [comment]
Here's a quick "top 3 items" we might want to consider for 2022:

- Refactoring for **maintainability** / ease-of-use & **reducing technical debt**: e.g.
  - #783
  -  Re-think & rework extension points and options
  - Consider aiming for a lightweight core + optional features
- **Evolution**: e.g.
  - Bootstrap 5 - #470
- **Semver** (#619): this is a big item (related to the first one). We might not have the resources necessary to make this happen yet.

**Comment by emckean** [comment]
Update from the Community meeting! It was wonderful to see so many community members in the call!
* PSC members introduced: @LisaFC, @geriom, @chalin, @emckean 
* Roadmap discussed: please add issues tagged 'roadmap' and make suggestions! 
* Versioning: plan is to move to VERSION 1, and have only one branch to better communicate when we make breaking changes
   * Related: we will begin instituting a CHANGELOG to record when features are added, dependencies updated, etc.

Please add in the comments anything I missed!

**Reply by deining** [reply]
> Update from the Community meeting! It was wonderful to see so many community members in the call!

I also enjoyed the meeting! Great to see that a lot of actions are taken to bring docsy further forward!

>     * Roadmap discussed: please add issues tagged 'roadmap' and make suggestions!

Great tool, I just added #470 to the proposed issues on the roadmap!

Are you aware of the fact that "ordinary" community members cannot add any labels (like `roadmap`) to issues?
I don't know how fine grained label assigment can be configured. Were great if we allowed community members to assign certain labels (like `roadmap`) to issues, thus keeping their motivation high.

**Comment by chalin** [comment]
As a possible alternative to using a `roadmap` label, I'd like to suggest that we use:

- [Milestones](https://github.com/google/docsy/milestones), I just created two: **22Q1** and **22Q2**. I find that planning by quarters works well for most projects.
- Priority [labels](https://github.com/google/docsy/labels) named **p**_x_, like [p1-high](https://github.com/google/docsy/labels/p1-high), etc., that I just added.

This would give folks a better idea of relative priority and planned timing for items.

Right, community members can't add tags. What they can do is vote for issues that are important to them and give feedback via discussions, etc. The PSC and docsy maintainers can then add appropriate tags and assign issues to milestones.

WDYT?

**Comment by emckean** [comment]
This sounds like a good solution to me :) Thanks @chalin!

**Comment by artberger** [comment]
Hi! I'm just dipping my toes into the Docsy community and trying to figure out the best ways to get plugged in. I found the [user group](https://groups.google.com/g/docsy-users), the Write the Docs #hugo Slack channel, and the Discussions in GitHub, and was wondering if there are still roadmap calls, working groups, or similar?

**Comment by emckean** [comment]
Hi Art! Thanks so much for your interest! The Project Steering Committee is working on the 2023 roadmap now, and I hope we'll be able to schedule a roadmap call this quarter. 

Some things you could absolutely help with in the meantime if you have bandwidth: 
- go through the large issue backlog and upvote ones that still seem relevant (extra bonus points if you are able to note issues that can be closed because they were fixed by subsequent PRs and note the PR number)
- test some of our installation instructions and note any conflicts/broken bits 

If there's any particular aspect of Docsy that interests you the most, let us know! Thank you!
